### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by ghost**
## Summary

- Footer

## Screenshots

Captura de pantalla de 2019-05-23 14-31-29

## Trello Card

https://trello.com/c/j2Pxstkl/14-footer

### CODE REVIEW

File: maquetado/scss/fonts.scss [hunk 1]
```diff
@@ -3,5 +3,7 @@ $nav-logo-height: 30px;
 $text-big: 24px;
 $text-medium: 16px;
 $text-small: 12px;
+$footer-text-big: 100px;
+$footer-height: 210px;
 $title-decoration-height: 90px;
 $title-decoration-width: 70px;
```
Comment (Dorian30) [inline maquetado/scss/fonts.scss]: Add an empty line

File: maquetado/scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  order: 0;
+  text-transform: uppercase;
+}
+
+.copyright {
+  align-self: flex-end;
+  color: $white;
+  margin-right: 30px;
+  order: 2;
+}
```
Comment (Dorian30) [inline maquetado/scss/footer.scss]: Same here

File: maquetado/scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  letter-spacing: 25px;
+  order: 1;
+  text-transform: uppercase;
+}
+
+.footer-text {
+  color: $white;
+  font-size: 27px;
+  order: 0;
+  text-transform: uppercase;
```
Comment (Dorian30) [inline maquetado/scss/footer.scss]: You can take this out and do 

```
footer-tittle,
footer-text {
  text-transform: uppercase;
}
```

Or you can define a class called `upper` and use it in every html tag you want to.

**Review by Dorian30 — CHANGES_REQUESTED** [review]

**Review by Dorian30 — APPROVED** [review]
Good work! :+1:

File: maquetado/index.html [hunk 4]
```diff
@@ -78,5 +78,10 @@ <h4 class="technology-release">Release:2005</h4>
       </div>
     </div>
   </div>
+  <div class="footer">
+    <h2 class="footer-title upper">wolox</h2>
```
Comment (mmamaniwolox) [inline maquetado/index.html]: This should be an asset, `wolox-white-logo` or something like that

File: maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  height: $footer-height;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.footer-title {
+  color: $white;
+  font-size: $footer-text-big;
+  letter-spacing: 25px;
+  order: 1;
```
Comment (mmamaniwolox) [inline maquetado/scss/footer.scss]: You dont need to add this, just order the elements in the index.html file

File: maquetado/scss/titles.scss [hunk 6]
```diff
@@ -1,6 +1,6 @@
 .title {
   font-size: $text-big;
-  margin: 50px 200px;
+  margin: 50px 200px 0;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: This margin is `margin: top left&right bottom`, are you sure you want this?

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/scss/footer.scss (hunk 5 above)
Comment (Dorian30) [reply maquetado/scss/footer.scss]: I think this is a good implementation since the idea behind css is to attack the layout without needing to do that through ordering tags in the html file.

File: maquetado/index.html [hunk 7]
```diff
@@ -78,5 +78,10 @@ <h4 class="technology-release">Release:2005</h4>
       </div>
     </div>
   </div>
+  <div class="footer">
+    <h3 class="footer-text upper">we are game changers</h3>
+    <h2 class="wolox-white-logo upper">wolox</h2>
```
Comment (mmamaniwolox) [inline maquetado/index.html]: This should be an `img` element with an asset

File: maquetado/scss/fonts.scss [hunk 8]
```diff
@@ -3,5 +3,7 @@ $nav-logo-height: 30px;
 $text-big: 24px;
 $text-medium: 16px;
 $text-small: 12px;
+$footer-text-big: 100px;
```
Comment (mmamaniwolox) [inline maquetado/scss/fonts.scss]: this should go in `footer.scss`

File: maquetado/scss/titles.scss (hunk 6 above)
Comment (mmamaniwolox) [reply maquetado/scss/titles.scss]: At least answer me haha

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/scss/titles.scss (hunk 6 above)
Comment (ArielFerraris) [reply maquetado/scss/titles.scss]: jajaja

File: maquetado/index.html [hunk 9]
```diff
@@ -78,5 +78,10 @@ <h4 class="technology-release">Release:2005</h4>
       </div>
     </div>
   </div>
+  <div class="footer">
+    <h3 class="footer-text upper">we are game changers</h3>
+    <img class="wolox-white-logo" src="./assets/wolox_logo.svg">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: Add a descriptive alt

File: maquetado/scss/footer.scss [hunk 10]
```diff
@@ -0,0 +1,32 @@
+$footer-text-big: 100px;
+$footer-height: 210px;
+
+.footer {
+  align-items: center;
+  background-color: #00ADEE;
```
Comment (mmamaniwolox) [inline maquetado/scss/footer.scss]: use `colors.css` and a constant

File: maquetado/scss/footer.scss [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+  display: flex;
+  flex-direction: column;
+  height: $footer-height;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.wolox-white-logo {
+  filter: brightness(100);
```
Comment (mmamaniwolox) [inline maquetado/scss/footer.scss]: you dont need this, there is a white logo asset, look for that one

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 12]
```diff
@@ -10,10 +10,10 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <nav class="navbar">
-    <img class="navbar-logo" src="./assets/wolox_logo.svg" alt="wolox logo"/>
+  <div class="navbar">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: it was ok as `nav`. Do not modify code from another PR here

File: maquetado/index.html [hunk 13]
```diff
@@ -24,25 +24,25 @@ <h4 class="technology-release">Release:2013</h4>
         <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/angular_icon.svg" alt="Angular"/>
+        <img class="logo" src="./assets/angular_icon.svg" alt="Angular">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: Why do you remove the closing label? it was ok with `/>`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]
